### PR TITLE
REL: 0.19

### PR DESCRIPTION
**Pull request by effigies**
I think this repository is in good shape to have a release, with Pythons 3.6-3.9 tested.

This PR does a little last minute cleanup, bumps the version, and uses the `README.md` as the PyPI description.

I'd appreciate a review and look over recent changes: https://github.com/python-versioneer/python-versioneer/compare/0.18...rel/0.19

Also, I would ask to be made a maintainer on PyPI. My username there is cjmarkie.

cc @bucknerns @yarikoptic @bashtage @warner

### CODE REVIEW

**Comment by effigies** [comment]
Anybody care to review?

File: setup.py [hunk 1]
```diff
@@ -2,18 +2,15 @@
...
-LONG="""
-Versioneer is a tool to automatically update version strings (in setup.py and
-the conventional 'from PROJECT import _version' pattern) by asking your
-version-control system about the current tree.
-"""
+LONG = Path.read_text(Path(__file__).parent / "README.md")
 
 # as nice as it'd be to versioneer ourselves, that sounds messy.
-VERSION = "0.19.dev0"
+VERSION = "0.19"
```
Comment (bashtage) [inline setup.py:13]: Slightly funny that versionner doesn't use versioneer to version.

**Comment by bashtage** [comment]
LGTM.  Should try it locally just to verify docs are all OK.

**Comment by bashtage** [comment]
Only thing I noticed: Should 0.19 be added to UPGRADING.md (under the same as 0.16,0.17 and 0.18)?

**Comment by effigies** [comment]
Thanks for the review. Added 0.19 to UPGRADING.

> LGTM. Should try it locally just to verify docs are all OK.

Not sure how to operationalize this. What were you thinking?

I have tested with:

```
pip install .
# go to some project
versioneer install
python setup.py version
```

**Comment by effigies** [comment]
Well, going to get this out. Here goes nothing.

**Pull request merged**